plugins/c8y_log_plugin/src/main.rs

```diff
@@ -126,6 +126,7 @@ async fn run(config_dir: impl AsRef<Path>, tedge_config: TEdgeConfig) -> Result<
         &mut mqtt_actor,
         &mqtt_schema,
         tedge_config.service.ty.clone(),
+        tedge_config.mqtt.topic_root.clone(),
```

Comment:
The same argument is given twice: first as an `MqttSchema`, second as `String`. Keep only the first one.

Reply:
fixed